src/components/views/messages/TextualBody.tsx

```diff
@@ -238,10 +238,15 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
     }
 
     private highlightCode(code: HTMLElement): void {
-        if (code.textContent.length > MAX_HIGHLIGHT_LENGTH) {
+        const codeTextContent = code.textContent;
+        if (!codeTextContent) {
+            console.log("Code block is empty");
```

Comment:
We use the loglevel library for logging, so no console.log statements, although I don't think a log line is necessary here - it could get quite noisy.

Reply:
there was a console.log when codeTextContent.length > MAX_HIGHLIGHT_LENGTH hence I thought same process should be replicated when exiting early for empty string / null

ref:
```javascript 
if (codeTextContent.length > MAX_HIGHLIGHT_LENGTH) {
    console.log(
        "Code block is bigger than highlight limit (" +
            codeTextContent.length +
            " > " +
            MAX_HIGHLIGHT_LENGTH +
            "): not highlighting",
    );
    return;
}
```